validator/validator/validator.py

```diff
@@ -216,7 +218,9 @@ def save_state(self):
             {
                 "step": self.step,
                 "hotkeys": self.hotkeys,
-                "scores": self.scores,
+                "baseline_averages": self.metrics.baseline_averages,
```

Comment:
This should probably be just one line, saving the state but excluding the metagraph. you can implement __getstate__ and __setstate__ in Metrics

Reply:
To clarify, it should save all the metrics together, to reduce future work.